test/CMakeLists.txt

```diff
@@ -14,9 +14,6 @@ find_package(benchmark)
 ##############################################################################
 # google test
 
-# remove export symbols from the cxx flags
-string(REPLACE "${EXPORTSYMBOLS}" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-
 # check for set environment variable
```

Comment:
I think I found a problem with this in another PR, I also have a solution there though.

Reply:
rebased this to 3.5.1 locally, will push it soon.